packages/rancher-pushprox/package.yaml

```diff
@@ -1,3 +1,3 @@
 url: local
-packageVersion: 01
-releaseCandidateVersion: 01
+packageVersion: 00
```

Comment:
reset to 00 since the major.minor.patch version was bumped. from now on, we should only ever be touching the releaseCandidateVersion for this package.